CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -445,7 +474,14 @@ def __init__(
             self.fridge_percentage: int = fridge_report.get("fridge_percentage", None)
 
     def set_timestamp(self):
-        self.timestamp = str(int(time.time()))
+        """
+        Sets epochTimestamp and timestamp fields
+        timestamp is ISO formatted date/time and is what the API user will use
+        epochTimestamp will be what is used to query the database
+        """
+        self.epochTimestamp = str(int(time.time()))
+        utc_time = datetime.datetime.utcnow()
+        self.timestamp = utc_time.strftime("%Y-%m-%dT%H:%M:%SZ")
```

Comment:
Why do we have a timestamp and an epoch time? Isn't epoch timestamp sufficient?

Reply:
The API users want iso formatted timestamps but db queries prefer epochTime. 

If we only did epochTimestamp we would have to convert it to ISO format on every request

Reply:
Personally, I think converting to ISO on each request would be easier to understand than having 2 timestamps in the database. But it would be slower for sure.

Reply:
Shouldn't any possible time conversion be done client side?

Reply:
I agree it would be easier to understand too

But we would have to do 3 steps in order to convert to ISO
1. Deserialize json_data
2. Convert to ISO
3. Serialize json_data

Reply:
Whenever a client requests data, wouldn't it process the JSON anyway?

Reply:
@abhinav1912 client side will be doing time conversion to the user's timezone. 

An API client would rather have the timestamp be in a human readable format (ISO format). It would also be nice for when someone if querying the database and can easily see the timestamps in ISO format

Reply:
@abhinav1912 Client side will have to deserialize the json data yes. The steps described are steps that we would have to do in the backend

Reply:
@0916dhkim @abhinav1912 would a field name change help at all?

epochTimestamp
isoTimestamp

Reply:
> would rather have the timestamp be in a human readable format (ISO format)

I've seen the engineering at 2-3 teams now, all of them use epoch timestamps over ISO ones. That being said, readability is a valid point, agreed.

Reply:
@abhinav1912 Looks like there's a split in opinion 😅 `"There is a split opinion between these two approaches. Twitter, DropBox, Segement.io APIs use ISO 8601, while Chargebee, Pusher and Stripe APIs use Unix Epoch Time."`

Here are some articles I read there are tradeoffs for both:

https://apiux.com/2013/03/20/5-laws-api-dates-and-times/ 
https://www.moesif.com/blog/technical/timestamp/manage-datetime-timestamp-timezones-in-api/

Reply:
> @0916dhkim @abhinav1912 would a field name change help at all?
> 
> epochTimestamp isoTimestamp

This is what I wanted to name it originally but I didn't want to have to change the API contract and have the frontend team have to create a PR to change their `timestamp` field to `isoTimestamp`. But it's a small enough change that we can do if the naming change would be helpful :)

Reply:
I don't really mind either naming or format as long as there is a single timestamp field.
If we are to go with the 2 timestamps approach, then some comments would be helpful.

Reply:
@0916dhkim I am still leaning towards having 2 timestamps. One for client and one for db so I'll add some comments

Reply:
Yes, sounds good to me :+1:

Reply:
done 646cfa3c0f570cb314a32521eedc099b8d321563